autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/reporting/remoteconfig/UnusedAutofillSiteBreakageReportingRemoteFeatureCodegenTrigger.kt

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2024 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.autofill.impl.reporting.remoteconfig
+
+import com.duckduckgo.anvil.annotations.ContributesRemoteFeature
+import com.duckduckgo.di.scopes.AppScope
+
+@ContributesRemoteFeature(
+    scope = AppScope::class,
+    boundType = AutofillSiteBreakageReportingFeature::class,
+    featureName = "autofillBreakageReporter",
+    settingsStore = AutofillSiteBreakageReportingRemoteSettingsPersister::class,
+    exceptionsStore = AutofillSiteBreakageReportingExceptionsPersister::class,
+)
+@Suppress("unused")
+private interface UnusedAutofillSiteBreakageReportingRemoteFeatureCodegenTrigger
```

Comment:
Why do we need this Unused interface if we can have the annotation in `AutofillSiteBreakageReportingFeature`?

Reply:
no good reason! removed. good suggestion!